README.md

```diff
@@ -1,57 +1,50 @@
-## Faros CI/CD GitHub Action
+# Faros CI/CD GitHub Action
 
-A GitHub Action to report CI/CD information for builds and deployments from a GitHub Action workflows to [Faros](https://www.faros.ai).
+A GitHub Action to report CI/CD information for code builds and deployments from a GitHub Action workflows to [Faros](https://www.faros.ai).
 
 ## Usage
 
 See [action.yml](action.yml) for the full documentation for this action's inputs and outputs.
 
-### Report a Build Event To Faros CI/CD Build Model
+### Report a code build (CI Event) To Faros
 
-To report a build event to Faros specify build in the `model` parameter and the build details.
+To report a code build to Faros specify `CI` in the `event` parameter and include the `CI` required fields.
 
 ```yaml
-- name: Report build info to Faros
-  id: emit-build-info
-  uses: faros-ai/faros-cicd-github-action@main
+- name: Report code build to Faros
+  id: send-ci-event
+  uses: faros-ai/faros-cicd-github-action@v2.0.0
```

Comment:
What version do we want to release this change under? `v2.0.0` or `v1.1.0`?

Reply:
Let's do v2.0.0 since they are technically breaking changes